src/operation/get-parameters-as-json-schema.js

```diff
@@ -517,13 +112,23 @@ function getParameters(path, operation, oas, globalDefaults) {
     const properties = parameters.reduce((prev, current) => {
       let schema = {};
       if ('schema' in current) {
+        const currentSchema = current.schema ? cloneObject(current.schema) : {};
+
+        if (current.example) {
+          // `example` can be present outside of the `schema` block so if it's there we should pull it in so it can be
+          // handled and returned if it's valid.
+          currentSchema.example = current.example;
+        } else if (current.examples) {
+          // `examples` isn't actually supported here in OAS 3.0, but we might as well support it because `examples` is
+          // JSON Schema and that's fully supported in OAS 3.1.
+          currentSchema.examples = current.examples;
+        }
```

Comment:
This block and the identical one a few lines down are the core fixes in this PR to support `example` and `examples` outside of `schema`. Everything else in this PR is just refactoring.